templates/workspace_detail.html

```diff
@@ -277,6 +276,20 @@ <h1 class="mb-2 text-3xl tracking-tight break-words font-bold text-slate-900 sm:
         {% /list_group %}
       {% /card %}
 
+      {% if analyses %}
+      {% #card title="Analyses" button=True button_text="View more" button_href=workspace.get_analyses_url %}
```

Comment:
```suggestion
      {% #card title="Interactive analyses" button=True button_text="View more" button_href=workspace.get_analyses_url %}
```
I wonder if people are going to get confused about what "Analyses" are, especially if they're moving between traditional OpenSAFELY projects and Interactive ones, so I think it would be a good idea to mention Interactive in the title. We could also add a subtitle explaining these were created using OpenSAFELY Interactive.